### DISCUSSION TITLE
Can’t connect to inverter

### DISCUSSION THREAD

**Discussion by jdmevo123** [WillCodeForCats/solaredge-modbus-multi#450] [General]
### Describe the bug

My inverter status in home assistant just show unavailable.

I noticed my configuration is setup different and not sure if I need to change it around or how to change it around. Please see screenshot of leader

### Expected behavior

Connection established

### Screenshots

![IMG_7539](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/6053506/4fe93c6e-abfa-43a4-ae4b-89a27f5b63b6)

### Diagnostic File

N/a

### Home Assistant Version

2013.10

### solaredge-modbus-multi Version

4.2

### Installation Type

HAOS

### Read the Instructions

- [X] I have read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki
- [X] My configuration follows https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration
- [X] This issue is not about a template error (template errors should not be reported as an issue, read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Design-Notes)

**Comment by WillCodeForCats** [comment]
I don't have any suggestions other than what I've already written in the wiki.

Maybe someone else will have some ideas.

**Reply by jdmevo123** [reply]
ive made a bit of progress, now I have values, but I cant use solar panel production daily in the energy dashboard.

When i go to the entity, it says it doesnt have a unique id and I cant mange it from the UI.

My energy.yaml configuration has the following:

utility_meter:
  solar_panel_production_daily:
    source: sensor.solar_panel_production_kwh
    name: Solar Panel Production Daily
    cycle: daily

**Reply by jdmevo123** [reply]
@WillCodeForCats i have found the issue i think, the template has sensor.solaredge_dc_power, but i do not have this entity, i only have the following:

sensor.solaredge_i1_dc_power
sensor.solaredge_i2_dc_power

How do i setup the one required?
